pyriemann/classification.py

```diff
@@ -498,19 +500,113 @@ def fit(self, X, y):
         return self
 
     def predict(self, covtest):
-        """get the predictions.
+        """Get the predictions.
 
         Parameters
         ----------
-        X : ndarray, shape (n_trials, n_channels, n_channels)
-            ndarray of SPD matrices.
+        X : ndarray, shape (n_matrices, n_channels, n_channels)
+            Set of SPD matrices.
 
         Returns
         -------
-        pred : ndarray of int, shape (n_trials, 1)
+        pred : ndarray of int, shape (n_matrices, 1)
             the prediction for each trials according to the closest centroid.
         """
         dist = self._predict_distances(covtest)
         neighbors_classes = self.classes_[np.argsort(dist)]
         out, _ = stats.mode(neighbors_classes[:, 0:self.n_neighbors], axis=1)
         return out.ravel()
+
+
+class SupportVectorMachine(BaseEstimator, ClassifierMixin):
```

Comment:
I would create RiemannSVC and RiemannSVR classes

Reply:
Good point. I don't like regression with SVMs but it should still be implemented.

Reply:
If a class `RiemannSVR` is added, it should be placed in a `regression` module, rather than `classification`.